db/pgresdb.js

```diff
@@ -42,3 +38,5 @@ client.connect();
 
 module.exports.query = queryDatabase;
```

Comment:
Can change to object for cleaniness
Example: 

module.exports = {
  queryDatabase,
  insert,
  findOnePlace,
  findReviews
};